docs/overrides/home.html

```diff
@@ -0,0 +1,188 @@
+{% extends "main.html" %}
+{% block tabs %}
+{{ super() }}
+<style>
```

Comment:
why is isn't in css ?

Reply:
This is due to the context of Material for MkDocs. For styles to be applied, they must be at the bottom of the cascading.